### PR TITLE
-ScanAvgLoadFactor

### PR DESCRIPTION
**Pull request by andrewjohnporter**
This is what I was informed by a Microsoft Support Engineer that Manually run scans will ignore this setting and run without any CPU limits

### CODE REVIEW

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -755,6 +755,8 @@ The default value is 50.
 
 Note: This is not a hard limit but rather a guidance for the scanning engine to not exceed this maximum on average.
 
+Note: Manually run scans will ignore this setting and run without any CPU limits
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:758]:
```suggestion
Note: Manually run scans will ignore this setting and run without any CPU limits.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Please insert a period as indicated.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]
It seems to be true. I tested it in two different machines, and this is what I saw:
image
image
For several minutes it stayed around those values.

**Comment by dariomws** [comment]
More evidence:
https://answers.microsoft.com/en-us/protect/forum/protect_defender-protect_scanning-windows_10/windows-10-full-scan-ignores-scanavgcpuloadfactor/3df45df7-6491-43f2-b85d-201b9d329e98
https://answers.microsoft.com/en-us/windows/forum/all/windows-defender-platform-418200610-uses-100-cpu/8dbd8e1f-e53f-471a-9e2f-9df6de65df2b

**Review by tseward — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by sdwheeler** [comment]
This change needs to be made for every version of the documentation that the change applies to. At a minimum, the same change should be made to the 2022 content. It may also apply to the 2016 content.

**Pull request merged**